velox/dwio/common/TimestampDecoder.h

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "velox/dwio/common/DirectDecoder.h"
+
+namespace facebook::velox::dwio::common {
+
+enum TIMESTAMP_PRECISION { MILLIS, MICROS };
```

Comment:
nit: TIMESTAMP_PRECISION -> TimestampPrecision and  Use kPascalCase for enumerators.
https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md#naming-conventions